packages/nx/src/command-line/release/changelog.ts

```diff
@@ -550,7 +552,32 @@ async function applyChangesAndExit(
         `No changes were detected for any changelog files, so no changelog entries will be generated.`,
       ],
     });
-    return;
+
+    if (!postGitTasks.length) {
+      // no GitHub releases to create so we can just exit
+      return;
+    }
+
+    if (isCI()) {
+      output.warn({
+        title: `Skipped GitHub release creation because no changes were detected for any changelog files.`,
+      });
+      return;
+    } else {
```

Comment:
Nit: the else is unnecessary with the early return

Reply:
removed